sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVSuite.scala

```diff
@@ -1012,6 +1012,196 @@ abstract class CSVSuite
     }
   }
 
+  test("SPARK-37326: Use different pattern to write and infer TIMESTAMP_NTZ values") {
```

Comment:
The test title says about different patterns but the patterns are the same in write and in inferring, in fact.

Reply:
Could you elaborate please? The code does use different timestamp format compared to the default one.

Reply:
The test title confuses because it can be read as the patterns in write and infer are different in the test.

Reply:
Let me rewrite this.